padinfo/padinfo.py

```diff
@@ -483,6 +484,31 @@ def get_id_emoji_options(self, m=None):
             self.settings.setEmojiServers(emoji_servers.split(','))
         await ctx.send(inline('Set {} servers'.format(len(self.settings.emojiServers()))))
 
+    @checks.is_owner()
+    @padinfo.command()
+    async def iddiff(self, ctx):
+        """Runs the diff checker for id and id2"""
+        await ctx.send("Running diff checker...")
+        hist_aggreg = list(self.historic_lookups) + list(self.historic_lookups_id2)
+        s = 0
+        f = []
+        for query in hist_aggreg:
```

Comment:
This will hang the bot. You should assume there are 20K lines in this file. I can provide the one from miru for you to test with.

Add a (tiny) sleep before every iteration. Avoid sending output to the context. Write results to a file (you can hardcode /tmp/diffcheck.csv).

Reply:
oh yeah, this _**will**_ hang the bot.  It's meant to only be used by testmiru or some other bot that shares data.  I don't have all the files in my miru, so i can only check out what's going on through yours.  It's not meant to be something that ever gets used after like a month from now